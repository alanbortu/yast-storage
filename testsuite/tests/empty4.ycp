// testedfiles: helper1b.yh
{
    include "setup-system.yh";

    setup_system("empty");

    include "helper1a.yh";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", true);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", true);
    ProductFeatures::SetBooleanFeature("partitioning", "vm_keep_unpartitioned_region", true);
    ProductFeatures::SetStringFeature("partitioning", "vm_desired_size", "30 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "20 GB");

    include "helper1b.yh";
}
