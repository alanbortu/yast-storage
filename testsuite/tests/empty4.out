Dump	Proposal:
Dump	Create boot volume /dev/sda1 (400.06 MiB) with ext4
Dump	Create volume /dev/sda2 (29.96 GiB)
Dump	Create volume group system (29.95 GiB) from /dev/sda2 [destructive]
Dump	Create logical volume /dev/system/home (19.45 GiB) for /home with ext4
Dump	Create logical volume /dev/system/root (10.00 GiB) for / with ext4
Dump	Create swap logical volume /dev/system/swap (512.00 MiB)
